src/agentscope/formatter/_truncated_formatter_base.py

```diff
@@ -36,13 +37,20 @@ def __init__(
                 The maximum number of tokens allowed in the formatted
                 messages. If not provided, the formatter will not truncate
                 the messages.
+            extract_image_blocks (`bool`, defaults to `False`):
```

Comment:
This argument should be exposed in the constructor of the corresponding API formatter class